app/forms/webauthn_setup_form.rb

```diff
@@ -45,13 +46,16 @@ def platform_authenticator?
 
   attr_reader :success, :transports, :invalid_transports
   attr_accessor :user, :challenge, :attestation_object, :client_data_json,
-                :name, :platform_authenticator
+                :name, :platform_authenticator, :authenticator_data_flags
 
   def consume_parameters(params)
     @attestation_object = params[:attestation_object]
     @client_data_json = params[:client_data_json]
     @name = params[:name]
     @platform_authenticator = (params[:platform_authenticator].to_s == 'true')
+    @authenticator_data_flags = process_authenticator_data_value(
+      params[:authenticator_data_value],
+    )
```

Comment:
Just flagging that we should expect this value will be validated as `nil` for some submissions during 50/50 deployment. I think that will be fine here with how we're rescuing `ArgumentError`.